temporal-sdk/src/main/java/io/temporal/client/UpdateWaitPolicy.java

```diff
@@ -23,7 +23,19 @@
 import io.temporal.api.enums.v1.UpdateWorkflowExecutionLifecycleStage;
 
 public enum UpdateWaitPolicy {
```

Comment:
Hmm in python looks like we changed the name to `WorkflowUpdateStage` I think we should do the same here because this enum will also be used when describing an updates stage.

Reply:
👍 And the docs here about what each enum means only apply to starting a workflow and should probably move to there.

Reply:
K, I'm down to change the name

Reply:
Docs wise, weirdly, even the proto APIs mention nothing about what the stages mean beyond as input to requests. That would be good to fix.